content/applications/sales/crm/performance/win_loss.rst

```diff
@@ -1,26 +1,702 @@
-=========================
-Check your Win/Loss Ratio
-=========================
+=================
+Pipeline Analysis
+=================
 
-To see how well you are doing with your pipeline, take a look at
-the Win/Loss ratio.
+The *CRM* app manages the sales pipeline as leads/opportunities move from stage to stage,
+origination to sale (Won) or archival (Lost).
 
-To access this report, go to your *Pipeline* view under the
-*Reporting* tab.
+After organizing the pipeline, use the search options and reports available on the *Pipeline
+Analysis* page to gain insight into the effectiveness of the pipeline and its users.
 
-From there you can filter to which opportunities you wish to see, yours,
-the ones from your sales channel, your whole company, etc. You can then
-click on filter and check Won/Lost.
+To access the *Pipeline Analysis* page, go to :menuselection:`CRM app --> Reporting --> Pipeline`.
 
-.. image:: win_loss/win_loss01.png
-    :align: center
+.. image:: win_loss/reporting-tab-and-pipeline-view.png
+   :align: center
+   :alt: Open the CRM app and click on the Reporting tab along the top, then click Pipeline.
 
-You can also change the *Measures* to *Total Revenue*.
+.. _win_loss/pipeline:
 
-.. image:: win_loss/win_loss02.png
-    :align: center
+Navigate the pipeline analysis page
+===================================
 
-You also have the ability to switch to a pie chart view.
+Upon accessing the :guilabel:`Pipeline Analysis` page, a bar graph showcasing the opportunities
+(active leads) from the past year automatically populates. The bars represent the number of
+opportunities in each stage of the sales pipeline, color-coded to show the month the opportunity
+reached that stage.
 
-.. image:: win_loss/win_loss03.png
-    :align: center
+.. image:: win_loss/pipeline-analysis-page.png
```

Comment:
[screenshot is not in HD](https://www.odoo.com/documentation/16.0/contributing/documentation/content_guidelines.html#screenshots), ideally the width caps to about 933px.

Double check the rest of the folder to make sure images are in HD and aren't too large in dimension. Remove any unnecessary screenshots or any that aren't used any more due to the rewrite.